pkg/components/rook-ceph/manifests.go

```diff
@@ -160,7 +160,7 @@ parameters:
 
   # Specify the filesystem type of the volume. If not specified, csi-provisioner
   # will set default as 'ext4'.
-  csi.storage.k8s.io/fstype: xfs
+  csi.storage.k8s.io/fstype: ext4
```

Comment:
Since we didn't use the default functionality of (`ext4` when not set), by explicitly setting it as `xfs`
How does this change effect existing rook ceph clusters based on `xfs` ?

What happens to existing `CephCluster` when we update the rook-ceph component ? 

Also is there any documentation on rook's end for migration path from `xfs` to `ext4` for existing CephClusters ?

Reply:
Why do we have to migrate anything? SC comes into picture only when someone creates a "new" PVC. So this setting is like changing the content on a stamp, so anything new will have new stamp content, old ones continue with what they have.

Reply:
I meant this change is in storage class resource.

So what happens if a PVC was created when the storage classes said the filesystem is xfs, now the same storage class is updated to ext4. Are there any implications to the existing PVC with the change or during backup restore of the PVC ?

Reply:
I don't think there are implications. Like the stamp example the PVC that were created with xfs will stay like that. Even for backup if it is with velero then the configs will be backed up so it should continue to work.